### PR TITLE
add elixir to quickstart tracing

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

File: content/quickstart/erlang/tracing.md [hunk 1]
```diff
@@ -237,6 +376,7 @@ And on clicking on `More info` we should see
 Resource|URL
 ---|---
 Zipkin project|https://zipkin.io/
-Erlang exporters|[Erlang exporters](/guides/exporters/supported-exporters)
+Erlang/Elixir exporters|[Erlang/Exliri exporters](/guides/exporters/supported-exporters)
```
Comment (odeke-em) [inline content/quickstart/erlang/tracing.md]: s/Exliri/Elixir/g

Comment (tsloughter) [reply content/quickstart/erlang/tracing.md]: Fixing

File: content/quickstart/erlang/tracing.md [hunk 2]
```diff
@@ -114,27 +159,46 @@ We will be tracing the execution as it starts in `read_eval_process/0`, goes to
...
 ].
-```
+{{</highlight>}}
+
+{{<highlight elixir>}}
+use Mix.Config
+
+config :opencensus,
+  sampler: {:oc_sampler_always, []},
+  reporter: {:oc_reporter_zipkin, []}
```
Comment (deadtrickster) [inline content/quickstart/erlang/tracing.md]: isn't it reporters now?

Comment (tsloughter) [reply content/quickstart/erlang/tracing.md]: Woops, yup, thought I fixed that already.

Comment (tsloughter) [reply content/quickstart/erlang/tracing.md]: fixed

**Pull request merged**